src/txn/txn.c

```diff
@@ -2506,23 +2508,38 @@ __wt_txn_global_shutdown(WT_SESSION_IMPL *session, const char **cfg)
          * clean shutdown.
          */
         if (use_timestamp) {
+            /* Initialize the tracking timer */
```

Comment:
I think the code would be clearer if we always timed it, rather than checking if we need to do any work.

I guess it could be confusing to see the "finished and ran for X seconds" line in a case where RTS didn't do any work, but that's what the other verbose messages are for.

Reply:
The main reason behind to find out the time calculation only when it is doing is because by default RTS verbose message are not enabled default, so it would be good with this message whether RTS has executed during shutdown or not? The same for the checkpoint also.

Reply:
OK, that seems reasonable - thanks for the explanation.